server/src/routes/category.js

```diff
@@ -0,0 +1,39 @@
+import express from "express";
+import { checkToken } from "../util/middleware.js";
+import * as validator from "../util/categoryMiddleware.js";
+import * as controller from "../controller/categoryController";
+import * as express_validator from "express-validator";
+
+const body = express_validator.body;
+const param = express_validator.param;
+
+const router = express.Router();
+// Guide: https://www.prisma.io/docs/concepts/components/prisma-client/crud
+
+
+router.post(
+    "/",
+    body("name", "Must include unique name for category").notEmpty(),
```

Comment:
maybe also do description body comment, even though not super necessary. 

body('description', 'Description is a string').optional().isString(),